packages/social-previews/src/nextdoor-preview/link-preview.tsx

```diff
@@ -15,6 +16,7 @@ export function NextdoorLinkPreview( props: NextdoorLinkPreviewProps ) {
 			// Override the props that are irrelevant to link preview
 			description=""
 			media={ undefined }
+			title={ props.title || stripHtmlTags( props.description ).substring( 0, 50 ) }
```

Comment:
I know it's not much, but do you think we should make this a helper function? If we want to change the length or something, we'll only have to change it in one place then.